infoschema/tables.go

```diff
@@ -1079,7 +1079,7 @@ var tableClusterInfoCols = []columnInfo{
 	{name: "STATUS_ADDRESS", tp: mysql.TypeVarchar, size: 64},
 	{name: "VERSION", tp: mysql.TypeVarchar, size: 64},
 	{name: "GIT_HASH", tp: mysql.TypeVarchar, size: 64},
-	{name: "START_TIME", tp: mysql.TypeVarchar, size: 32},
+	{name: "START_TIME", tp: mysql.TypeDatetime, size: -1},
```

Comment:
It looks like for `TypeDatetime` we inconsistently specify the size as: `-1`, `19`, or omit it.

This doesn't introduce any new inconsistencies, but it would be nice to get an answer on what is considered most correct. @wjhuang2016 do you have ideas?

Reply:
19 (which ignores fraction) or 26 (which uses the max fraction) is ok.

Reply:
Is this something that should be in https://pingcap.github.io/tidb-dev-guide/ or not?

Reply:
I don't think so.